crates/driver/src/infra/blockchain/mod.rs

```diff
@@ -27,7 +28,8 @@ impl Rpc {
     /// at the specifed URL.
     pub async fn new(url: &url::Url) -> Result<Self, Error> {
```

Comment:
Please rename the constructor to `try_new()` to follow Rust coding convention.

Reply:
There is a lot of similar constructors that are returning `Result`. I think we can do this in a separate PR, it could be a good first issue.

https://github.com/cowprotocol/services/issues/3179